src/integration-test/tests/citizen/defence/steps/defence.ts

```diff
@@ -367,6 +371,46 @@ export class DefenceSteps {
     I.see('You’ve submitted your response')
   }
 
+  partialPaymentNotMade (defendantType: PartyType, paymentOption: PaymentOption): void {
+    I.see('Have you paid the claimant the amount you admit you owe?')
+    haveYouPaidTheClaimantPage.selectNoOption()
+    defendantTaskListPage.selectTaskHowMuchMoneyBelieveYouOwe()
+    defendantHowMuchYouOwePage.enterAmountOwed(10)
+    defendantTaskListPage.selectTaskWhyDoYouDisagreeWithTheClaim()
+    defendantYourDefencePage.enterYourDefence('random text')
+    this.addTimeLineOfEvents(defence.timeline)
+    this.enterEvidence('description', 'They do not have evidence')
+    defendantTaskListPage.selectTaskDecideHowWillYouPay()
+    switch (paymentOption) {
+      case PaymentOption.IMMEDIATELY:
+        defendantWhenWillYouPage.chooseImmediately()
+        break
+      case PaymentOption.BY_SET_DATE:
+        defendantWhenWillYouPage.chooseFullBySetDate()
+        defendantPaymentDatePage.enterDate('2025-01-01')
+        defendantPaymentDatePage.saveAndContinue()
+        defendantTaskListPage.selectShareYourFinancialDetailsTask()
+        statementOfMeansSteps.fillStatementOfMeansWithMinimalDataSet()
+        break
+      case PaymentOption.INSTALMENTS:
+        defendantWhenWillYouPage.chooseInstalments()
+        defendantTaskListPage.selectYourRepaymentPlanTask()
+        defendantPaymentPlanPage.enterRepaymentPlan(defendantRepaymentPlan)
+        defendantPaymentPlanPage.saveAndContinue()
+        defendantTaskListPage.selectShareYourFinancialDetailsTask()
+        statementOfMeansSteps.fillStatementOfMeansWithFullDataSet()
+        break
+      default:
+        throw new Error(`Unknown payment option: ${paymentOption}`)
+    }
+
+    defendantTaskListPage.selectTaskFreeMediation()
+    defendantFreeMediationPage.chooseNo()
```

Comment:
In future pass in Free Mediation Choice, so when we get to Claimant Response Test can act accordingly